imap_data_access/io.py

```diff
@@ -159,6 +163,85 @@ def query(
             "At least one query parameter must be provided. "
             "Run 'query -h' for more information."
         )
+    # Check instrument name
+    if instrument is not None:
+        if instrument not in (
```

Comment:
Can you get this from `VALID_INSTRUMENTS` https://github.com/IMAP-Science-Operations-Center/imap-data-access/blob/main/imap_data_access/__init__.py